docs/Brew-Livecheck.md

```diff
@@ -75,63 +75,81 @@ We sometimes make this more explicit to exclude unwanted matches. URLs with a pr
 ### Version directories
 
 ```ruby
-  livecheck do
-    url "https://www.example.com/releases/example/"
-    regex(%r{href=["']?v?(\d+(?:\.\d+)+)/?["' >]}i)
-  end
+livecheck do
+  url "https://www.example.com/releases/example/"
+  regex(%r{href=["']?v?(\d+(?:\.\d+)+)/?["' >]}i)
+end
 ```
 
 When checking a directory listing page, sometimes files are separated into version directories (e.g. `1.2.3/`). In this case, we must identify versions from the directory names.
 
 ### Git tags
 
 ```ruby
-  livecheck do
-    url :stable
-    regex(/^v?(\d+(?:\.\d+)+)$/i)
-  end
+livecheck do
+  url :stable
+  regex(/^v?(\d+(?:\.\d+)+)$/i)
+end
 ```
 
 When the `stable` URL uses the `Git` strategy, the regex above will only match tags like `1.2`/`v1.2`, etc.
 
 If tags include the software name as a prefix (e.g. `example-1.2.3`), it's easy to modify the regex accordingly: `/^example[._-]v?(\d+(?:\.\d+)+)$/i`
 
-### `PageMatch` `strategy` block
+### `strategy` blocks
+
+If the upstream version format needs to be manipulated to match the formula/cask format, a `strategy` block can be used instead of a `regex`. 
```

Comment:
As mentioned in [my comment on the docs PR](https://github.com/Homebrew/brew/pull/10329#issuecomment-760332137), I don't think it should be an either/or situation and I specifically added the ability for us to pass the `regex` into `strategy` blocks for a good reason.

In instances where we're only using one regex in the `strategy` block, it allows us to continue to surface the regex in the debug and JSON output, which is useful. When the regex isn't specified, it will also pass in a regex that's generated within the strategy (i.e., the strategy regex), so we can simply use that in the block instead of having to create a regex literal to replicate it.

There are benefits to specifying the regex outside of the strategy block but I don't see any benefit to always specifying the regex(es) within the strategy block. It unnecessarily prevents us from surfacing that information in a useful manner in the output.